reactor-core/src/main/java/reactor/core/publisher/MonoCacheTime.java

```diff
@@ -84,6 +91,11 @@
 		this.state = emptyState;
 	}
 
+	@Override
+	public Context currentContext() {
+		return null;
```

Comment:
NEVER return null. IDEA should have warned you there that this is a NonNull method/API.

Reply:
Oh wow, I missed this auto-generated one! Nice catch!